scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -69,6 +70,9 @@ workflow GvsExtractCallset {
   String fq_ranges_cohort_vet_extract_table_name = "~{full_extract_prefix}__VET_DATA"
   String fq_ranges_cohort_vet_extract_table = "~{fq_cohort_dataset}.~{fq_ranges_cohort_vet_extract_table_name}"
 
+  # make the fully qualified version of the ploidy table if present, otherwise leave it undefined
+  String? fq_ploidy_mapping_table = if (defined(ploidy_table_name)) then "~{fq_gvs_dataset}.~{ploidy_table_name}" else ploidy_table_name
```

Comment:
I *think* the way WDL works is that if the optional `ploidy_table_name` is undefined then this entire expression will be undefined, like [here](https://github.com/broadinstitute/gatk/blob/1d17d17681cebe6cd02497d4fb4fffe6a21cac1b/scripts/variantstore/wdl/GvsExtractCallset.wdl#L408)

```suggestion
  String? fq_ploidy_mapping_table = ~{fq_gvs_dataset + "." + ploidy_table_name}
```

Reply:
Hmm.  I looked at the [wdl spec](https://github.com/openwdl/wdl/blob/wdl-1.1/SPEC.md#concatenation-of-optional-values) for a bit on this one when writing it, and it _seemed_ to say that copying the pattern above would result in a "." and not undefined if either was undefined.  The rules about when things become the empty string and when they stay undefined vs turning into the empty string are something I need to test more.

But if I bring the dot inside of the evaluation brackets, I _should_ be able to get the effect that I need.

Reply:
so I tried this several times again, and WDL just doesn't let you use a construct like that when assigning a variable.  The IDE throws errors and it fails womtool validation.  I don't know of a cleaner way right now to take a String? and turn it into an interpolated String?.  The main thing WDL wants to do is turn it into an empty string, and that isn't the behavior I desire here (because I want the entire line of arguments to GATK to disappear below if the argument isn't passed, and that won't happen if it's just an empty string but WILL if it's still an unset optional)

Reply:
Ugh. Well unless any of our WDL experts (like @gbggrant) have suggestions here I guess don't worry about it...